site/content/en/reference/_index.md

```diff
@@ -93,6 +93,53 @@ By default, kpt will use the builtin schema.
   The path to an OpenAPI schema file. The default value is ./openapi.json
 ```
 
+### Annotations
+
+Kpt supports a small set of annotations that allow customizing the behavior
+of kpt on a per-resource basis
+
+#### config.kubernetes.io/local-config
+
+config.kubernetes.io/local-config declares that the configuration is to local
+tools rather than a resource that should be applied to an APIServer. Function
+configuration defined declaratively in KRM files is an example of a resource
+that should contain this annotation so that tools know it is not intended to
+be sent to the Kubernetes api server.
+
+Example:
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  annotations:
+    config.kubernetes.io/function: |
+      container:
+        image: gcr.io/kpt-functions/label-namespace
+    config.kubernetes.io/local-config: "true"
```

Comment:
Does the value matter? Must it parse as a true Boolean (per [`strconv.ParseBool`](https://golang.org/pkg/strconv/#ParseBool)), or is the value immaterial and the annotation's presence enough of an indicator?